rasterio/rio/warp.py

```diff
@@ -135,8 +135,7 @@ def warp(
                         raise click.BadParameter('Required when using --bounds',
                             param='res', param_hint='res')
 
-                    xmin, ymin, xmax, ymax = transform_bounds(src.crs, dst_crs,
-                                                              *bounds)
+                    xmin, ymin, xmax, ymax = bounds
```

Comment:
Definitely a good fix for now. I wonder if we should consider an option like gdalwarp's `-te_srs` to define the srs of the bounds. Example use case: reprojecting from utm to mercator but only know the desired bounds in latlong.